src/test/obj_pool_open_mt/TESTS.py

```diff
@@ -0,0 +1,29 @@
+#!../env.py
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2021, Intel Corporation
+#
+
+import testframework as t
+
+
+class BASE(t.BaseTest):
+    test_type = t.Medium
+    niter = 2
+
+    def run(self, ctx):
+        ctx.exec('obj_pool_open_mt', ctx.testdir, self.niter)
+
+
+@t.require_valgrind_enabled('helgrind')
+class TEST0(BASE):
+    "multi-threaded open test"
+
+
+@t.require_valgrind_enabled('helgrind')
```

Comment:
What is the diff between TEST0 and TEST1 ?

Reply:
i meant to only add valgrind to TEST1. done.